components/eam/cime_config/config_component.xml

```diff
@@ -60,7 +60,6 @@
       <!-- they will be removed in a future PR -->
       <value compset="_EAM%SP[12]V1"        >-use_MMF -crm_adv MPDATA -nlev 72 -crm_nz 58 -crm_dx 1000 -crm_dt 5 </value>
       <value compset="_EAM%SP[12]V1"        >-microphys mg2  </value>
-      <value compset="_EAM%SP[12]V1"        >-cppdefs ' -DMMF_DIR_NS ' </value>
```

Comment:
Can you remind me why we are still keeping  EAM%SP compsets? No one seems to be using them.

Reply:
It was for some sense of backwards compatibility, but I agree with you sentiment that they aren't useful anymore. I'll make a note to make another PR to remove them.